api/common.js

```diff
@@ -415,14 +415,11 @@ exports.load_github_detail = function(service_name, cb) {
 
 exports.generateQuery = function(str){
-    let fil = str.toLowerCase()
-    .split(' ').
-    filter((word) => !stopwords.includes(word)).
-    map((word) => `W='${word}'`).join(',');
-    return "OR("+fil+")"
+    stopwords.add('');
```

Comment:
why are you adding an empty string here? also.. this function gets called for every project, so if you want to update the stopwords, you should do it right after you import this library?

Reply:
I am sorry I will fix it right away.